.travis.yml

```diff
@@ -1,9 +1,18 @@
-sudo: false
+sudo: true
 services:
   - redis-server
 language: ruby
 rvm:
   - '2.3'
 before_install:
-  - cd ruby
   - gem install bundler -v 1.5.1
+  - cd python
+  - sudo pip install -U pip 
+  - sudo pip install setuptools==33.1.1
+  - pip --version
+  - sudo pip install -r requirements.txt
```

Comment:
Yeah, I enqueued a couple dozen builds to get this working. There is likely a sudo free version that would work, but I don't know enough about all that stack to not spend days on it.